src/components/WalletConnect/theme.js

```diff
@@ -0,0 +1,71 @@
+import { StyleSheet } from 'react-native';
+import { COLORS } from '../../styles/themes';
+
+export const commonStyles = StyleSheet.create({
+  // Card
+  card: {
+    paddingVertical: 16,
+    paddingHorizontal: 16,
+    backgroundColor: COLORS.backgroundColor,
+    borderTopLeftRadius: 16,
+    borderTopRightRadius: 16,
+    borderBottomLeftRadius: 16,
+    borderBottomRightRadius: 16,
```

Comment:
You right, we can use `borderRadius` but I prefer to be explicit, especially because we can add variations in the card component over time and the borders may become asymmetric.